database/migrations/2024_10_11_180406_create_flights_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Lightit\Backoffice\Airlines\Domain\Models\Airline;
+use Lightit\Backoffice\Cities\Domain\Models\City;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('flights', function (Blueprint $table) {
+            $table->id();
+
+            $table->foreignIdFor(Airline::class)
+                ->constrained()
+                ->cascadeOnDelete();
+            $table->foreignIdFor(City::class, 'origin_city_id')
+                ->constrained('cities')
+                ->cascadeOnDelete();
+            $table->foreignIdFor(City::class, 'destination_city_id')
+                ->constrained('cities')
+                ->cascadeOnDelete();
+
+            $table->timestamp('departure_time');
+            $table->timestamp('arrival_time');
```

Comment:
```suggestion
            $table->timestamp('departure_at');
            $table->timestamp('arrival_at');
```
While this might seem like something small, these suffixes are not interchangeable:
- time
- date
- at

`departure_time` = **time** of departure, i.e: 20:45:00
`departure_date` = **date** of departure, i.e: 2024-01-01
`departure_at` = **date + time** of departure, i.e: 2024-01-01 20:45:00

A flight leaves on a certain date at a certain time, meaning the right suffix for these columns is `at`.